tools/topology/sof/pipe-waves-codec-playback.m4

```diff
@@ -0,0 +1,38 @@
+# Low Latency Passthrough with Waves codec Pipeline and PCM
+#
+# Pipeline Endpoints for connection are :-
+#
+# host PCM_P --> B0 --> Codec Adapter --> B1 --> sink DAI0
+
+# Waves codec setup config
+define(`CA_SETUP_CONTROLBYTES',
+``      bytes "0x53,0x4f,0x46,0x00,'
+`       0x00,0x00,0x00,0x00,'
+`       0x20,0x00,0x00,0x00,'
+`       0x00,0x10,0x00,0x03,'
+`       0x00,0x00,0x00,0x00,'
+`       0x00,0x00,0x00,0x00,'
+`       0x00,0x00,0x00,0x00,'
+`       0x00,0x00,0x00,0x00,'
+
+`       0x00,0x01,0x41,0x57,'
+`       0x00,0x00,0x00,0x00,'
+`       0x80,0xBB,0x00,0x00,'
+`       0x20,0x00,0x00,0x00,'
+`       0x02,0x00,0x00,0x00,'
+
+`       0x00,0x00,0x00,0x00,'
+`       0x0c,0x00,0x00,0x00,'
+`       0x00,0x00,0x00,0x00"''
+)
+define(`CA_SETUP_CONTROLBYTES_MAX', 8192)
+define(`CA_SETUP_CONTROLBYTES_NAME', `MaxxChrome Setup ')
+
+# use default runtime config but change max size
+define(`CA_RUNTIME_CONTROLBYTES_MAX', 8192)
+define(`CA_RUNTIME_CONTROLBYTES_NAME', `MaxxChrome Runtime ')
```

Comment:
what is the name of the controls created.

Just wondering if the pipeline ID will be used for symmetry?

From a user perspective, having a 'speaker' and 'headphone' settings would be more intuitive than pipeline IDs...

Reply:
Agreed, but would have to pass that in from the top level m4 as at this stage you are are a generic pipeline

Reply:
I am not saying no, I am just saying this is not the file we should define it, only the receiver of the names.  So yes i am ok with passing names through vars like WoV.